packages/affiliate/tests/php/test_Affiliate.php

```diff
@@ -1,23 +1,20 @@
 <?php
-/**
- * Tests for Jetpack_Affiliate
- */
+use Automattic\Jetpack\Partners\Affiliate;
 
-use Automattic\Jetpack\JITM\Manager as JITM_Manager;
+use PHPUnit\Framework\TestCase;
 
 // Load required class to get the affiliate code
 require_once JETPACK__PLUGIN_DIR . 'class.jetpack.php';
```

Comment:
Having this as a requirement for testing indicates that this package isn't independent enough to be a package of its own. Ideally, it should be able to live and run outside of Jetpack, and all of its dependencies should be defined explicitly in its `composer.json`.